tests/utils/factories.py

```diff
@@ -271,47 +415,69 @@ class DatasetDataFactory(factory.DictFactory):
     created_by = factory.Faker('uuid4')
     updated_by = factory.Faker('uuid4')
     deleted_by = factory.Faker('uuid4')
-    unique_name = None
-    create_time = None
-    update_time = None
-    delete_time = None
+    unique_name = None  # TODO Update tests to include unique_name
+    # Unfortunately, this does not respect chronology
+    update_time = factory.Faker("unix_milliseconds")
+    create_time = factory.Faker("unix_milliseconds")
+    delete_time = factory.Faker("unix_milliseconds")
     public = False
 
 
 class IDDataFactory(factory.DictFactory):
     id = factory.Faker('uuid4')
 
 
-class LinkByUIDInputFactory(factory.DictFactory):
-    id = factory.Faker('uuid4')
-    type = 'link_by_uid'
+class LinkByUIDFactory(factory.Factory):
+    class Meta:
+        model = LinkByUID
+
     scope = 'id'
+    id = factory.Faker('uuid4')
 
 
-class FileLinkDataFactory(factory.DictFactory):
+class FileLinkFactory(factory.Factory):
+    class Meta:
+        model = FileLink
+
     url = factory.Faker('uri')
     filename = factory.Faker('file_name')
-    type = 'file_link'
 
 
-class MaterialRunDataFactory(factory.DictFactory):
+class ProcessTemplateFactory(factory.Factory):
```

Comment:
Still doesn't have `audit_info` attached, which is pretty important since we create citrine-python child class with extra fields from an upstream library.